package.json

```diff
@@ -2,8 +2,9 @@
   "name": "omnicalculator-eslint-config",
   "engines": {
     "node": "^20.12.0",
-    "pnpm": "^9.4.0"
+    "pnpm": "^10.5.0"
   },
+  "packageManager": "pnpm@10.5.0+sha512.11106a5916c7406fe4b8cb8e3067974b8728f47308a4f5ac5e850304afa6f57e2847d7950dfe78877d8d36bfb401d381c4215db3a4c3547ffa63c14333a6fa51",
```

Comment:
You can try bumping that to the latest minor just before merging, right now it's 10.5.2.